lib/gcloud/jsondoc/markup_helper.rb

```diff
@@ -30,12 +30,20 @@ def linkify(name, title)
         code_obj = YARD::Registry.resolve object.namespace, name, true
         if code_obj.nil?
           name
+        elsif code_obj.type == :method
+          path = code_obj.namespace.path.gsub("::", "/").downcase
+          method_type = get_method_type code_obj
+          method_id = "#{code_obj.name}-#{method_type}"
```

Comment:
I was trying to hold closely to prior art such as rDoc and YARD. Yard for example: [Array#count-instance_method](http://www.rubydoc.info/stdlib/core/Array#count-instance_method) and [Hash#try_convert-class_method](http://www.rubydoc.info/stdlib/core/Hash#try_convert-class_method). But most importantly, if we just use words, we never have to worry about which symbol to use (or not use). It's very simple and human-readable.